firebase-dataconnect/src/test/kotlin/com/google/firebase/dataconnect/serializers/TimestampSerializerUnitTest.kt

```diff
@@ -75,6 +75,23 @@ class TimestampSerializerUnitTest {
       .isEqualTo(Timestamp(1136214245, 123_456_789))
   }
 
+  @Test
```

Comment:
Please add some tests where there is the +/- time zone AND 3, 6, and 9 digits of nanoseconds.